build.sbt

```diff
@@ -20,84 +26,156 @@ def commonProject(id: String): Project = {
   Project(id, file(id)).settings(
     name := id,
 
+    // Include all of the dependencies in the loader. The base loader will be the Application
+    // ClassLoader. All classes apart from system classes will be reloaded with each run instead
+    // of being cached between layers.
+    classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.Flat,
+
     scalacOptions ++= Seq(
-      "-Xfatal-warnings",
-      "-deprecation:false",
+      "-deprecation",
       "-feature",
+      "-Xfatal-warnings",
       "-Ywarn-dead-code",
       "-encoding", "UTF-8"
     ),
 
-    ivyScala := ivyScala.value map {
-      _.copy(overrideScalaVersion = true)
-    },
-
     // don't publish the test code as an artifact anymore, since we have playJsonTests
-    publishArtifact in Test := false,
+    Test / publishArtifact := false,
 
     // disable compilation of ScalaDocs, since this always breaks on links
-    sources in(Compile, doc) := Seq.empty,
+    Compile / doc / sources := Seq.empty,
 
     // disable publishing empty ScalaDocs
-    publishArtifact in(Compile, packageDoc) := false
+    Compile / packageDoc / publishArtifact := false
 
   ).enablePlugins(SemVerPlugin)
 }
 
-def playJsonOps(includePlayVersion: String): Project = {
-  val playSuffix = includePlayVersion match {
-    case Dependencies.play23Version => "23"
-    case Dependencies.play25Version => "25"
-  }
-  val scalatestVersion = includePlayVersion match {
-    case Dependencies.play23Version => Dependencies.scalatest2Version
-    case Dependencies.play25Version => Dependencies.scalatest3Version
-  }
-  val id = s"play$playSuffix-json-ops"
+def playSuffix(includePlayVersion: String): String = includePlayVersion match {
+  case Play_2_5 => "25"
+  case Play_2_6 => "26"
+  case Play_2_7 => "27"
+}
+
+def scalaVersions(includePlayVersion: String): Seq[String] = includePlayVersion match {
+  case Play_2_5 => Seq(Scala_2_11)
+  case Play_2_6 => Seq(Scala_2_12, Scala_2_11)
+  case Play_2_7 => Seq(Scala_2_13)
```

Comment:
```suggestion
  case Play_2_7 => Seq(Scala_2_11 , Scala_2_13)
```

Reply:
Unfortunately, these versions cannot be cross-compiled. I would have to create a separate Scala 2.11 version.

Reply:
Fixed.